platform/viewer/src/lib/DICOMSR/retrieveDataFromMeasurements.js

```diff
@@ -0,0 +1,52 @@
+import * as dcmjs from 'dcmjs';
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+
+const retrieveDataFromMeasurements = measurements => {
+  const cornerstoneAdpater = dcmjs.adapters.Cornerstone;
+  const { MeasurementReport } = cornerstoneAdpater;
+  const { getImageIdForImagePath } = OHIF.measurements;
+
+  const toolState = {};
+  const unsupportedTools = [];
+
+  Object.keys(measurements).forEach(measurementType => {
+    const annotations = measurements[measurementType];
+
+    annotations.forEach(annotation => {
+      const { toolType, imagePath } = annotation;
```

Comment:
Won't only measurements from existing SRs have an imagePath? Or is this something our MeasurementsApi has been tacking on and I've never noticed?